src/app/components/Statistics/DateRange/DateRange.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { forwardRef } from 'react';
+import DatePicker from 'react-datepicker';
+import './custompicker.css';
+import styled from 'styled-components';
+import moment from 'moment';
+import TextField from '@mui/material/TextField';
+import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
+import { Button } from '@mui/material';
+import 'react-datepicker/dist/react-datepicker.css'
+
+const PickerContainer = styled.div`
+  display: flex;
+  width: 250px;
+`;
+const Container = styled.div`
+  display: flex;
+  justify-content: space-around;
+`;
+const FilterContainer = styled.div`
+  display: flex;
+`;
+const StyledButton = styled(Button)`
+  margin-top: 0rem;
+  margin-bottom: 1rem;
+  margin-right: 1rem;
+  margin-left: 1rem;
+`;
+const NewTextField = styled(TextField)`
+  margin: 1rem;
+  width: 125px;
+`;
+const CustomInputContainer = styled.div`
+  display: flex;
+  align-content: center;
+  align-items: baseline;
+  cursor: pointer;
+`;
+
+const DateRange = ({
+  initialStartDate,
+  InitianEndDate,
+  setEndDate,
+  setStartDate,
+  darkMode
+}) => {
+  const changeDate = date => {
+    setEndDate(new Date());
+    setStartDate(new Date(date));
+  };
+  console.log({
```

Comment:
also here

Reply:
@Ayagoumi solved conflicts and removed logs